litgpt/__main__.py

```diff
@@ -107,6 +108,20 @@ def main() -> None:
             subsubcommand_parser.add_function_arguments(v["fn"])
             subcommands.add_subcommand(k, subsubcommand_parser, help=v["help"])
 
+    # CLI specific defaults
+    root_parser.set_defaults(
+        {
+            "finetune.qlora.quantize": "bnb.nf4-dq",
+            "finetune.qlora.precision": "bf16-true",
+            "finetune.qlora.lora_query": True,
```

Comment:
They are not necessary, but the QLoRA paper sets them.

LitGPT implements the original LoRA release by default which only sets queries and values. There were discussions in the past to change this but nothing came out of it.

I thought that since we are adding a qlora-specific command, it would make sense to use the settings in the paper.

Reply:
I see. Sounds good then.